Tofu/AccountsViewController.swift

```diff
@@ -123,6 +135,26 @@ class AccountsViewController: UITableViewController {
             navigationItem.leftBarButtonItem = editButtonItem
         }
     }
+    
+    // MARK: UIImagePickerControlDelegate
+    
+    func imagePickerController(_ picker: UIImagePickerController,
+                               didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
+        guard let selectedQRCode = info[UIImagePickerController.InfoKey.originalImage] as? UIImage,
+            let detector = CIDetector(ofType: CIDetectorTypeQRCode,
+                                      context: nil,
+                                      options: [CIDetectorAccuracy: CIDetectorAccuracyHigh]),
+            let ciImage = CIImage(image: selectedQRCode),
+            let features = detector.features(in: ciImage) as? [CIQRCodeFeature] else { return }
+
+        let qrCodeURL = URL(string: features.reduce("") { $0 + ($1.messageString ?? "") })
```

Comment:
Did you ever see more then one feature returned? We seem to be able to provide a [CIDetectorMaxFeatureCount](https://developer.apple.com/documentation/coreimage/cidetectormaxfeaturecount) option to specify how many we want at most. The docs do however say this defaults to 1.

If so, what do you think about including this in the guard statement? Maybe something like this:

```
guard ...,
    let features = detector.features(in: ciImage) as? [CIQRCodeFeature],
    let messageString = features.first?.messageString,
    let qrCodeURL = URL(string: messageString) else { return }

Reply:
I did not, but I agree that including it in the guard is a good idea.